tests/integration/targets/lookup_random_pet/test.yml

```
@@ -0,0 +1,10 @@
+- hosts: localhost
+  tasks:
+
+  - assert:
+      that:
+      - query('community.general.random_pet', words=3) | length == 1
+      - query('community.general.random_pet', length=3) | length == 1
+      - query('community.general.random_pet', prefix="kuberentes") | length == 1
+      - query('community.general.random_pet', separator="_") | length == 1
+      - query('community.general.random_pet', words=2, length=6, prefix="kuberentes", separator="_") | length == 1
```

```suggestion
  - name: Call plugin
    set_fact:
      result1: '{{ query('community.general.random_pet', words=3) }}'
      result2: '{{ query('community.general.random_pet', length=3) }}'
      result3: '{{ query('community.general.random_pet', prefix="kuberentes") }}'
      result4: '{{ query('community.general.random_pet', separator="_") }}'
      result5: '{{ query('community.general.random_pet', words=2, length=6, prefix="kuberentes", separator="_") }}'

  - name: Check results
    assert:
      that:
        - result1 | length == 1
        - result2 | length == 1
        - result3 | length == 1
        - result4 | length == 1
        - result5 | length == 1
```
Now it's easier to add more tests, like checking whether `result1[0]` actually has three words etc.